portfolio/src/main/webapp/scriptMainPage.js

```diff
@@ -0,0 +1,52 @@
+function onClickGithub() {
+    window.open("https://github.com/thomaspr1205");
+}
+
+function onClickLinkedin() {
+    window.open("https://www.linkedin.com/in/thomaspenar");
+}
+
+function onClickProject3() {
+    document.getElementById("project3").style.display = "block";
+}
+
+async function showRandomQuestion() {
+    const responseFromServer = await fetch('/chat');
+    const questionList = await responseFromServer.json();
+
+    const questionsContainer = document.getElementById('question-container');
+    randomIndex = Math.floor(Math.random() * 9) + 0;
+    randomQuestion = questionList[randomIndex];
+    questionsContainer.innerText = randomQuestion;
+}
+
+// Accordions
```

Comment:
Comment should reflect what the function is doing such that anyone reading this comment can immediately understand what the function does.